kombu/transport/azureservicebus.py

```diff
@@ -92,7 +93,10 @@ def _put(self, queue, message, **kwargs):
     def _get(self, queue, timeout=None):
         """Try to retrieve a single message off ``queue``."""
         message = self.queue_service.receive_queue_message(
-            self.entity_name(queue), timeout=timeout, peek_lock=False)
+            self.entity_name(queue),
+            timeout=timeout or self.wait_time_seconds,
```

Comment:
instead of the `or` clause, you could inline the usage and not cache anything:

```py
if timeout is None:
  timeout = self.transport_options.get('wait_time_seconds',
                                          self.default_wait_time_seconds)

Reply:
this attribute will be accessed many times, once per message, the cache gets some gain from it and other implementations follow this pattern